source/leadership.rst

```diff
@@ -0,0 +1,133 @@
+red member chooses to change focus, that change of focus will be announced at
+the next monthly team meeting. Red group members are expected to also perform
+duties of blue group members while facilitating and helping the blue group to
+grow.
+
+**Blue Group** (includes all subproject deputies and yt project members) – This
+is where the journey of a new yt project team member starts. Blue group members
+spend a large part of your time facilitating contributions from others to the
+project (e.g. reviewing work, answering questions, maintaining project
+infrastructure, doing triage of new issues). We expect blue members to teach
+and grow contributors that can become blue members. This encourages diversity
+of the project. After building a good understanding of the technology and
+social structure of the project blue members can change to a red member by
+volunteering for tasks which a red member is expected to perform. After a while
+of doing so the group will ask if you want to become a red member and, if you
+agree, recognise you as such at a monthly team meeting.
+
+**Green Group** (can be any person, any time, self-nominated only)-- This group
+applies to members of the community whose life situation has changed so that
+they prefer to (temporarily) not take on the rights and responsibilities of
+being an active member. Membership to the green group is invoked by
+self-nomination. They can return to active membership by resuming their
+community activities. Their return will be announced at the next team meeting
+or via the developers mailing list.
+
+**Emeritus Group** (time based or self-nominated)-- this group is for members
+of the community at any level that have a prolonged leave of member duties from
+the yt project. We welcome participation in the community for any length of
+time, and this is a way to retire from the project. A project member may become
+emeritus by self-nomination or by not interacting in a membership capacity on
+the yt project for over 2 years. After this time, a member of the executive
+committee will contact an inactive member and see if they wish to remain active
+or transition to an emeritus role. 
+
+Finally, members of the community that wish to not be associated with the yt
+project may request to be removed from any/all teams/groups/committees and
+documentation by e-mailing either confidential@yt-project.org or
+yt-exec@googlegroups.com.
+
+Steering Committee
+------------------
+
+Members are eligible to serve on the yt project steering committee (also
+sometimes referred to as the executive committee). The
+steering committe shall meet quarterly or more often as needed to discuss
+project-related business. Members may join the steering committee by attending
+one of the team meetings and self-nominating. Once nominated, a nominee's entry
+to the steering committee will be voted on by members of the steering committee
+by a vote ratio equal or exceeding to 3/5. One person on the steering
```

Comment:
The 3/5 ratio doesn't play great with low numbers like we'll have on the SC. Perhaps majority+1?

Reply:
I expect that most perspective members will be allowed on unanimously and we'd want to add people via consensus. 3/5 seems fine to me.